crowdloans/rewards/polkadot.json

```diff
@@ -1,4 +1,83 @@
 {
+  "2019": {
+    "cDot": "${amount * 1}",
+    "reward": {
+      "total": "5% of Total Supply",
+      "fixed": [{ "value": 0.48, "suffix": "+", "unit": "LAYR" }],
+      "unit": "LAYR",
+      "detail": [
+        {
+          "label": "Est. Project Rewards",
+          "formula": "${amount * 0.48}"
+        },
+        {
+          "label": "Early Bird Bonus",
+          "formula": [
+            {
+              "condition": { "rangeDate": ["12/15/21", "12/15/21"] },
+              "value": "${amount * .48 * 0.15}"
+            },
+            {
+              "condition": { "rangeDate": ["12/16/21", "12/17/21"] },
```

Comment:
the end date is 12/18/21

Reply:
-5% bonus to those who staked in the first 24 hours, ending 12/15/21 at 9PM EST
-10% bonus to stakers in the first three days of opening the crowdloan, ending 12/18/21 at 9PM EST

2019, Composable Finance

Reply:
i think it should be 15% bonus for the first 24 hrs that should be incentive right more bonus for early investing? @Helen8

Reply:
in notion it is 5%. I was also confused.